MyKkumi/MyKkumi/Presentation/Home/HomeViewController.swift

```diff
@@ -25,81 +26,45 @@ class HomeViewController: BaseViewController {
         searchView.addSubview(searchButton)
         view.addSubview(notificationButton)
         view.addSubview(shoppingCartButton)
-        view.addSubview(banner)
-        view.addSubview(bannerPage)
         view.addSubview(makePost)
+        view.addSubview(postTableView)
     }
     
-    public override func setupBind() {
-        let dataSource = RxCollectionViewSectionedReloadDataSource<SectionModel<String, BannerVO>>(configureCell: { (_, collectionView, indexPath, bannerVO) -> UICollectionViewCell in
-            self.updateIndex()
-            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: BannerCollectionCell.cellID, for: indexPath) as! BannerCollectionCell
-            if let urlString = bannerVO.imageURL {
-                cell.imageView.load(url: URL(string: urlString)!, placeholder: "placeholder")
-            }
-            return cell
-        })
-        viewModel.banners
-            .map { [SectionModel(model: "Section 1", items: $0)] }
-            .asDriver(onErrorJustReturn: [])
-            .drive(banner.rx.items(dataSource: dataSource))
-            .disposed(by: disposeBag)
-
-        banner.rx.modelSelected(BannerVO.self)
-            .subscribe(onNext : {[weak self] bannerVO in
-                if let id = bannerVO.id {
-                    self?.viewModel.bannerTap.onNext(id)
-                }
+    public override func setupBind(viewModel : HomeViewModelProtocol) {
+        super.setupBind(viewModel: viewModel)
+        
+        self.viewModel = viewModel
+        
+        self.rx.viewDidLoad.bind(to: viewModel.viewdidload).disposed(by: disposeBag)
```

Comment:
```suggestion
        self.rx.viewDidLoad
            .bind(to: viewModel.viewdidload)
            .disposed(by: disposeBag)
```

인덴트 최대한 일관성있게 맞춰주세요.